### PR TITLE
Add new setting to messaging-policies-in-teams.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8522.

### CODE REVIEW

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

File: Teams/messaging-policies-in-teams.md [hunk 1]
```diff
@@ -80,7 +81,7 @@ Here are the messaging policy settings that you can configure.
 <a name="bkchat"> </a>
 
 - **Chat**  Turn this setting on if you want users in your organization to be able to use the Teams app to chat with other people.
```
Comment (JohanFreelancer9) [inline Teams/messaging-policies-in-teams.md:83]:
```suggestion
- **Chat** Turn this setting on if you want users in your organization to be able to use the Teams app to chat with other people.
```

File: Teams/messaging-policies-in-teams.md [hunk 2]
```diff
@@ -80,7 +81,7 @@ Here are the messaging policy settings that you can configure.
...
-- *Use Giphy in conversations**  If you turn on Giphys, users can include Giphys in chat conversations with other people. Giphy is an online database and search engine that allows users to search for and share animated GIF files. Each Giphy is assigned a content rating. In addition to turning on this setting, you need to enable [Optional Connected Experiences](/deployoffice/privacy/manage-privacy-controls#policy-setting-for-optional-connected-experiences) to allow Giphys in conversations.
+- **Use Giphy in conversations**  If you turn on Giphys, users can include Giphys in chat conversations with other people. Giphy is an online database and search engine that allows users to search for and share animated GIF files. Each Giphy is assigned a content rating. In addition to turning on this setting, you need to enable [Optional Connected Experiences](/deployoffice/privacy/manage-privacy-controls#policy-setting-for-optional-connected-experiences) to allow Giphys in conversations.
```
Comment (JohanFreelancer9) [inline Teams/messaging-policies-in-teams.md:84]:
```suggestion
- **Use Giphy in conversations** If you turn on Giphys, users can include Giphys in chat conversations with other people. Giphy is an online database and search engine that allows users to search for and share animated GIF files. Each Giphy is assigned a content rating. In addition to turning on this setting, you need to enable [Optional Connected Experiences](/deployoffice/privacy/manage-privacy-controls#policy-setting-for-optional-connected-experiences) to allow Giphys in conversations.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@dariomws could you please help on the copy review? Thanks!

**Pull request closed without merging**